sdks/go/pkg/beam/core/runtime/exec/plan.go

```diff
@@ -24,6 +24,7 @@ import (
 	"time"
 
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/internal/errors"
+	"google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
Is timestamppb the type we already represent watermarks as internally? From a cursory look it feels weird to represent it with a protobuf type, but I'm also not sure what we currently use so I just want to check.

Reply:
It is in several places, but most importantly in our OutputWatermarks proto - https://github.com/apache/beam/blob/81d3de2007f82703d93c420bf03b00d00c03e96e/sdks/go/pkg/beam/model/fnexecution_v1/beam_fn_api.pb.go#L922